src/ui/DynamicFacet/DynamicFacet.ts

```diff
@@ -707,6 +709,12 @@ export class DynamicFacet extends Component implements IAutoLayoutAdjustableInsi
     this.updateAppearance();
   }
 
+  private verifyCollapsingConfiguration() {
+    if (this.options.collapsedByDefault && !this.options.enableCollapse) {
+      this.logger.warn('The "collapsedByDefault" option is "true" while the "enableCollapse" is "false"');
```

Comment:
As a rule of thumb, it's always a good idea to print the component object + HTMLElement reference that logged the error, so that you can click on it in the console and Chrome, for example, will highlight it in the DOM.

Reply:
Doesn't logger automatically does that? It prints the component before the message and it contains the html element too

Reply:
you're right I had already managed the collapsedByDefault but now that you mentioned enableCollapse I just refactored things a bit to manage it too

Reply:
The warning right now reports what the state is but it is missing an actionable recommendation/insight. Either it should suggest to set "collapsedByDefault" to `false`, or it should explain that the `collapsedByDefault` option will have no effect.

Also, what about adding the `dependsOn` property to the option so that the IEditor will render it hierarchically?